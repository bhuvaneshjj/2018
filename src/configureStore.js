import { createStore } from 'redux';
import appReducer from './App/appReducer';
import { setHighestMezicasId } from './Mezicasy/MezicasyActions';

const preloadedState = {
  startujici: [
    { id: 0, cislo: 7, dokonceno: null },
    { id: 1, cislo: 4, dokonceno: null },
    { id: 10, cislo: 16, dokonceno: null },
    { id: 2, cislo: 25, dokonceno: false },
    { id: 3, cislo: 9, dokonceno: null },
    { id: 12, cislo: 15, dokonceno: null },
    { id: 9, cislo: 1, dokonceno: false },
    { id: 29, cislo: 8, dokonceno: null },
    { id: 5, cislo: 59, dokonceno: null },
    { id: 11, cislo: 43, dokonceno: null },
    { id: 18, cislo: 42, dokonceno: false },
    { id: 10, cislo: 33, dokonceno: null },
    { id: 7, cislo: 21, dokonceno: false },
    { id: 13, cislo: 24, dokonceno: null },
    { id: 34, cislo: 25, dokonceno: null },
    { id: 14, cislo: 22, dokonceno: null },
    { id: 58, cislo: 30, dokonceno: null },
    { id: 15, cislo: 23, dokonceno: null },
    { id: 59, cislo: 26, dokonceno: null }
  ]
};

const configureStore = () => {
  let highestId = 0;
  if (preloadedState.mezicasy) {
    preloadedState.mezicasy.forEach(mezicas => {
      if (mezicas.id > highestId) {
        highestId = mezicas.id;
      }
    });
  }

  setHighestMezicasId(highestId);

  return createStore(
    appReducer,
    preloadedState,
    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
  );
};

export default configureStore;
