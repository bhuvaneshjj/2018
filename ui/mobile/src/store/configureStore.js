import AsyncStorage from 'react-native'; // :TODO: use @react-native-community/async-storage
import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import wsAPI from '../ui-common/store/wsAPI';
import { websocketConnected, websocketDisconnected } from '../ui-common/App/connectedActions';
import appReducer from '../App/appReducer';

export const loadState = async () => {
  try {
    const serializedState = await AsyncStorage.getItem('state');
    if (serializedState === null || serializedState === undefined) {
      return undefined;
    }
    return JSON.parse(serializedState);
  } catch (err) {
    // :TODO: think about how to report the problem
    // eslint-disable-next-line no-console
    console.log('Problem while loading app state from the async storage', err);
    return undefined;
  }
};

const saveState = async state => {
  try {
    const serializedState = JSON.stringify(state);
    await AsyncStorage.setItem('state', serializedState);
  } catch (err) {
    // :TODO: think about how to report the problem
    // eslint-disable-next-line no-console
    console.log('Problem while storing app state to the async storage', err);
  }
};

const setupWsClient = (wsClient, store) => {
  wsClient.setCallbacks({
    onBroadcast: ({ broadcast }) => {
      switch (broadcast) {
        /* :TODO:
        case BROADCAST_UCASTNIK:
          store.dispatch(broadcastUcastnik(data));
          break;
        case BROADCAST_STOPKY:
          store.dispatch(broadcastStopky(data));
          break;
        */
        default:
          break;
      }
    },
    onConnect: async () => {
      await store.dispatch(websocketConnected());
    },
    onClose: () => store.dispatch(websocketDisconnected())
  });

  try {
    /* Start asynchronous connect to the websocket server.
     WsClient will retry indefinitely if the connection cannot be established. */
    wsClient.connect();
  } catch (err) {
    // Silently ignore any errors. They should have been dispatched from WsClient anyway.
  }
};

const configureStore = (wsClient, initialStateParam) => {
  const initialState = initialStateParam || {};

  // eslint-disable-next-line no-underscore-dangle
  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
  const store = createStore(
    appReducer,
    initialState,
    composeEnhancers(
      applyMiddleware(thunk.withExtraArgument(wsClient), wsAPI.withExtraArgument(wsClient))
    )
  );

  store.subscribe(async () => {
    const state = store.getState();
    await saveState({
      auth: {
        authenticated: state.auth.authenticated,
        decodedToken: state.auth.decodedToken,
        token: state.auth.token
      },
      serviceURL: state.serviceURL
    });
  });

  if (wsClient) {
    setupWsClient(wsClient, store);
  }

  return store;
};

export default configureStore;
