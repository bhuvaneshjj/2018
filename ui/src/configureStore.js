import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import appReducer from './App/appReducer';
import { websocketConnected, websocketDisconnected } from './App/AppActions';
import { setHighestMezicasId } from './casomeric/Mezicasy/MezicasyActions';

const demoStartujiciState = {
  startujici: [
    { id: 0, cislo: 7, dokonceno: null },
    { id: 1, cislo: 4, dokonceno: null },
    { id: 17, cislo: 16, dokonceno: null },
    { id: 2, cislo: 25, dokonceno: false },
    { id: 3, cislo: 9, dokonceno: null },
    { id: 12, cislo: 15, dokonceno: null },
    { id: 9, cislo: 1, dokonceno: false },
    { id: 29, cislo: 8, dokonceno: null },
    { id: 5, cislo: 59, dokonceno: null },
    { id: 11, cislo: 43, dokonceno: null },
    { id: 18, cislo: 42, dokonceno: false },
    { id: 10, cislo: 33, dokonceno: null },
    { id: 7, cislo: 21, dokonceno: false },
    { id: 13, cislo: 24, dokonceno: null },
    { id: 34, cislo: 27, dokonceno: null },
    { id: 14, cislo: 22, dokonceno: null },
    { id: 58, cislo: 30, dokonceno: null },
    { id: 15, cislo: 23, dokonceno: null },
    { id: 59, cislo: 26, dokonceno: null }
  ]
};

const loadState = () => {
  try {
    const serializedState = localStorage.getItem('state');
    if (serializedState === null) {
      return undefined;
    }
    return JSON.parse(serializedState);
  } catch (err) {
    console.log('Problem while loading app state from the local storage', err);
    return undefined;
  }
};

const saveState = state => {
  try {
    const serializedState = JSON.stringify(state);
    localStorage.setItem('state', serializedState);
  } catch (err) {
    console.log('Problem while storing app state to the local storage', err);
  }
};

const setupWsClient = (wsClient, store) => {
  wsClient.setCallbacks({
    onConnect: () => {
      store.dispatch(websocketConnected());
    },
    onClose: () => store.dispatch(websocketDisconnected())
  });

  try {
    /* Start asynchronous connect to the websocket server.
     WsClient will retry indefinitely if the connection cannot be established. */
    wsClient.connect();
  } catch (err) {
    // Silently ignore any errors. They should have been dispatched from WsClient anyway.
  }
};

const configureStore = (wsClient, initialStateParam = loadState()) => {
  const initialState = initialStateParam || {};
  initialState.startujici = demoStartujiciState.startujici;

  if (initialState.casomeric && initialState.casomeric.mezicasy) {
    let highestId = 0;

    initialState.casomeric.mezicasy.forEach(mezicas => {
      if (mezicas.id > highestId) {
        highestId = mezicas.id;
      }
    });

    setHighestMezicasId(highestId);
  }

  // eslint-disable-next-line no-underscore-dangle
  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
  const store = createStore(
    appReducer,
    initialState,
    composeEnhancers(applyMiddleware(thunk.withExtraArgument(wsClient)))
  );

  store.subscribe(() => {
    const state = store.getState();
    saveState({
      auth: {
        authenticated: state.auth.authenticated,
        decodedToken: state.auth.decodedToken,
        token: state.auth.token
      },
      casomeric: { stopky: state.casomeric.stopky, mezicasy: state.casomeric.mezicasy }
    });
  });

  if (wsClient) {
    setupWsClient(wsClient, store);
  }

  return store;
};

export default configureStore;
