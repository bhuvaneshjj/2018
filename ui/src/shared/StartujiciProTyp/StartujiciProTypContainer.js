import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import {
  decorareStartujici,
  getStartCislaProTyp,
  getStartujiciProTyp
} from './startujiciProTypReducer';
import StartujiciProTyp from './StartujiciProTyp';

const mapStateToProps = state => state;

const mergeProps = (stateProps, dispatchProps, ownProps) => {
  const { entities } = stateProps;
  const { jenStartujici, prihlasky, typ, onClick } = ownProps;

  const startujici = jenStartujici
    ? getStartujiciProTyp({ prihlasky, typ, ...entities })
    : getStartCislaProTyp({ prihlasky, typ, ...entities });

  const decorated = jenStartujici
    ? decorareStartujici(startujici, () => true, onClick)
    : decorareStartujici(startujici, jedenStartujici => !jedenStartujici.id, onClick);
  return {
    startujici: decorated
  };
};

const StartujiciProTypContainer = connect(mapStateToProps, null, mergeProps)(StartujiciProTyp);

StartujiciProTypContainer.propTypes = {
  jenStartujici: PropTypes.bool.isRequired,
  prihlasky: PropTypes.bool.isRequired,
  typ: PropTypes.string.isRequired,
  onClick: PropTypes.func
};

export default StartujiciProTypContainer;
