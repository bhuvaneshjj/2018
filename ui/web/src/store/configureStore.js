import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import { BROADCAST_STOPKY, BROADCAST_UCASTNIK } from 'ui-common/common';
import wsAPI from 'ui-common/store/wsAPI';
import appReducer from '../App/appReducer';
import { websocketConnected, websocketDisconnected } from '../App/AppActions';
import { broadcastStopky } from '../entities/stopky/stopkyActions';
import { broadcastUcastnik } from '../entities/ucastnici/ucastniciActions';
import { timesyncOperation, timesyncStart } from '../Timesync/TimesyncActions';

const loadState = () => {
  try {
    const serializedState = localStorage.getItem('state');
    if (serializedState === null) {
      return undefined;
    }
    return JSON.parse(serializedState);
  } catch (err) {
    // :TODO: think about how to report the problem
    // eslint-disable-next-line no-console
    console.log('Problem while loading app state from the local storage', err);
    return undefined;
  }
};

const saveState = state => {
  try {
    const serializedState = JSON.stringify(state);
    localStorage.setItem('state', serializedState);
  } catch (err) {
    // :TODO: think about how to report the problem
    // eslint-disable-next-line no-console
    console.log('Problem while storing app state to the local storage', err);
  }
};

const setupWsClient = (wsClient, store) => {
  wsClient.setCallbacks({
    onBroadcast: ({ broadcast, data }) => {
      switch (broadcast) {
        case BROADCAST_UCASTNIK:
          store.dispatch(broadcastUcastnik(data));
          break;
        case BROADCAST_STOPKY:
          store.dispatch(broadcastStopky(data));
          break;
        default:
          break;
      }
    },
    onConnect: async () => {
      await store.dispatch(websocketConnected());
      store.dispatch(timesyncOperation());
    },
    onClose: () => store.dispatch(websocketDisconnected())
  });

  try {
    /* Start asynchronous connect to the websocket server.
     WsClient will retry indefinitely if the connection cannot be established. */
    wsClient.connect();
  } catch (err) {
    // Silently ignore any errors. They should have been dispatched from WsClient anyway.
  }
};

const configureStore = (wsClient, initialStateParam = loadState()) => {
  const initialState = initialStateParam || {};

  // eslint-disable-next-line no-underscore-dangle
  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
  const store = createStore(
    appReducer,
    initialState,
    composeEnhancers(
      applyMiddleware(thunk.withExtraArgument(wsClient), wsAPI.withExtraArgument(wsClient))
    )
  );

  store.subscribe(() => {
    const state = store.getState();
    saveState({
      auth: {
        authenticated: state.auth.authenticated,
        decodedToken: state.auth.decodedToken,
        token: state.auth.token
      }
      // :TODO: casomeric.mezicasy
    });
  });

  // The time synchronization gets started by default.
  store.dispatch(timesyncStart());

  if (wsClient) {
    setupWsClient(wsClient, store);
  }

  return store;
};

export default configureStore;
