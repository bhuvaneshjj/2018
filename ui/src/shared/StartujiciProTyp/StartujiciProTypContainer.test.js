import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import ucastniciTestData from '../../entities/ucastnici/ucastniciTestData';
import StartujiciProTypContainer from './StartujiciProTypContainer';

const mockStore = configureStore();

let store;
beforeEach(() => {
  const state = ucastniciTestData;
  store = mockStore(state);
  store.dispatch = jest.fn();
});

it('maps state to props - půlmaraton, startovní čísla, přihlášky', () => {
  const wrapper = shallow(
    <StartujiciProTypContainer
      jenStartujici={false}
      odstartovani={false}
      typ="půlmaraton"
      renderer={jest.fn()}
      store={store}
    />
  );
  expect(wrapper.props().startujici).toBeTruthy();
  expect(wrapper.props().startujici).toMatchSnapshot();
});

it('maps state to props - půlmaraton, startující, přihlášky', () => {
  const wrapper = shallow(
    <StartujiciProTypContainer
      jenStartujici={true}
      odstartovani={false}
      typ="půlmaraton"
      renderer={jest.fn()}
      store={store}
    />
  );
  expect(wrapper.props().startujici).toBeTruthy();
  expect(wrapper.props().startujici).toMatchSnapshot();
});

it('maps state to props - půlmaraton, startovní čísla, odstartováni', () => {
  const wrapper = shallow(
    <StartujiciProTypContainer
      jenStartujici={false}
      odstartovani={true}
      typ="půlmaraton"
      renderer={jest.fn()}
      store={store}
    />
  );
  expect(wrapper.props().startujici).toBeTruthy();
  expect(wrapper.props().startujici).toMatchSnapshot();
});

it('maps state to props - půlmaraton, startující, odstartováni', () => {
  const wrapper = shallow(
    <StartujiciProTypContainer
      jenStartujici={true}
      odstartovani={true}
      typ="půlmaraton"
      renderer={jest.fn()}
      store={store}
    />
  );
  expect(wrapper.props().startujici).toBeTruthy();
  expect(wrapper.props().startujici).toMatchSnapshot();
});
