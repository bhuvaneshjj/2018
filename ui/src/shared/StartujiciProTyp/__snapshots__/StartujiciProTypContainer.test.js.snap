// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maps state to props - půlmaraton, startovní čísla, odstartováni 1`] = `
Array [
  Object {
    "startCislo": 1,
  },
  Object {
    "startCislo": 2,
  },
  Object {
    "startCislo": 3,
  },
  Object {
    "startCislo": 4,
  },
  Object {
    "startCislo": 5,
  },
  Object {
    "startCislo": 6,
  },
  Object {
    "startCislo": 7,
  },
  Object {
    "startCislo": 8,
  },
  Object {
    "startCislo": 9,
  },
  Object {
    "startCislo": 10,
  },
  Object {
    "cas": "PT2H6M32.6S",
    "dokonceno": true,
    "id": "7a09b1fd371dec1e99b7e142",
    "startCislo": 11,
  },
  Object {
    "startCislo": 12,
  },
  Object {
    "startCislo": 13,
  },
  Object {
    "startCislo": 14,
  },
  Object {
    "dokonceno": false,
    "id": "5a09b1fd371dec1e99b7e1c9",
    "startCislo": 15,
  },
  Object {
    "startCislo": 16,
  },
  Object {
    "startCislo": 17,
  },
  Object {
    "startCislo": 18,
  },
  Object {
    "startCislo": 19,
  },
  Object {
    "startCislo": 20,
  },
  Object {
    "startCislo": 21,
  },
  Object {
    "startCislo": 22,
  },
  Object {
    "startCislo": 23,
  },
  Object {
    "startCislo": 24,
  },
  Object {
    "startCislo": 25,
  },
  Object {
    "startCislo": 26,
  },
  Object {
    "startCislo": 27,
  },
  Object {
    "startCislo": 28,
  },
  Object {
    "startCislo": 29,
  },
  Object {
    "startCislo": 30,
  },
  Object {
    "startCislo": 31,
  },
  Object {
    "startCislo": 32,
  },
  Object {
    "startCislo": 33,
  },
  Object {
    "startCislo": 34,
  },
  Object {
    "startCislo": 35,
  },
  Object {
    "startCislo": 36,
  },
  Object {
    "startCislo": 37,
  },
  Object {
    "startCislo": 38,
  },
  Object {
    "startCislo": 39,
  },
  Object {
    "startCislo": 40,
  },
  Object {
    "startCislo": 41,
  },
  Object {
    "startCislo": 42,
  },
  Object {
    "startCislo": 43,
  },
  Object {
    "startCislo": 44,
  },
  Object {
    "startCislo": 45,
  },
  Object {
    "startCislo": 46,
  },
  Object {
    "startCislo": 47,
  },
  Object {
    "startCislo": 48,
  },
  Object {
    "startCislo": 49,
  },
  Object {
    "startCislo": 50,
  },
  Object {
    "startCislo": 51,
  },
  Object {
    "startCislo": 52,
  },
  Object {
    "startCislo": 53,
  },
  Object {
    "startCislo": 54,
  },
  Object {
    "startCislo": 55,
  },
  Object {
    "startCislo": 56,
  },
  Object {
    "startCislo": 57,
  },
  Object {
    "startCislo": 58,
  },
  Object {
    "startCislo": 59,
  },
  Object {
    "startCislo": 60,
  },
  Object {
    "startCislo": 61,
  },
  Object {
    "startCislo": 62,
  },
  Object {
    "startCislo": 63,
  },
  Object {
    "startCislo": 64,
  },
  Object {
    "startCislo": 65,
  },
  Object {
    "startCislo": 66,
  },
  Object {
    "startCislo": 67,
  },
  Object {
    "startCislo": 68,
  },
  Object {
    "startCislo": 69,
  },
  Object {
    "startCislo": 70,
  },
  Object {
    "startCislo": 71,
  },
  Object {
    "startCislo": 72,
  },
  Object {
    "startCislo": 73,
  },
  Object {
    "startCislo": 74,
  },
  Object {
    "startCislo": 75,
  },
  Object {
    "startCislo": 76,
  },
  Object {
    "startCislo": 77,
  },
  Object {
    "startCislo": 78,
  },
  Object {
    "startCislo": 79,
  },
  Object {
    "startCislo": 80,
  },
  Object {
    "startCislo": 81,
  },
  Object {
    "startCislo": 82,
  },
  Object {
    "startCislo": 83,
  },
  Object {
    "startCislo": 84,
  },
  Object {
    "startCislo": 85,
  },
  Object {
    "startCislo": 86,
  },
  Object {
    "startCislo": 87,
  },
  Object {
    "startCislo": 88,
  },
  Object {
    "startCislo": 89,
  },
  Object {
    "startCislo": 90,
  },
  Object {
    "startCislo": 91,
  },
  Object {
    "startCislo": 92,
  },
  Object {
    "startCislo": 93,
  },
  Object {
    "startCislo": 94,
  },
  Object {
    "startCislo": 95,
  },
  Object {
    "startCislo": 96,
  },
  Object {
    "startCislo": 97,
  },
  Object {
    "startCislo": 98,
  },
  Object {
    "startCislo": 99,
  },
  Object {
    "startCislo": 100,
  },
]
`;

exports[`maps state to props - půlmaraton, startovní čísla, přihlášky 1`] = `
Array [
  Object {
    "startCislo": 1,
  },
  Object {
    "startCislo": 2,
  },
  Object {
    "startCislo": 3,
  },
  Object {
    "startCislo": 4,
  },
  Object {
    "startCislo": 5,
  },
  Object {
    "startCislo": 6,
  },
  Object {
    "startCislo": 7,
  },
  Object {
    "startCislo": 8,
  },
  Object {
    "startCislo": 9,
  },
  Object {
    "id": "7a09b1fd371dec1e99b7e142",
    "startCislo": 10,
  },
  Object {
    "startCislo": 11,
  },
  Object {
    "startCislo": 12,
  },
  Object {
    "startCislo": 13,
  },
  Object {
    "startCislo": 14,
  },
  Object {
    "startCislo": 15,
  },
  Object {
    "startCislo": 16,
  },
  Object {
    "id": "5a09b1fd371dec1e99b7e1c9",
    "startCislo": 17,
  },
  Object {
    "startCislo": 18,
  },
  Object {
    "startCislo": 19,
  },
  Object {
    "startCislo": 20,
  },
  Object {
    "startCislo": 21,
  },
  Object {
    "startCislo": 22,
  },
  Object {
    "startCislo": 23,
  },
  Object {
    "startCislo": 24,
  },
  Object {
    "startCislo": 25,
  },
  Object {
    "startCislo": 26,
  },
  Object {
    "startCislo": 27,
  },
  Object {
    "startCislo": 28,
  },
  Object {
    "startCislo": 29,
  },
  Object {
    "startCislo": 30,
  },
  Object {
    "startCislo": 31,
  },
  Object {
    "startCislo": 32,
  },
  Object {
    "startCislo": 33,
  },
  Object {
    "startCislo": 34,
  },
  Object {
    "startCislo": 35,
  },
  Object {
    "startCislo": 36,
  },
  Object {
    "startCislo": 37,
  },
  Object {
    "startCislo": 38,
  },
  Object {
    "startCislo": 39,
  },
  Object {
    "startCislo": 40,
  },
  Object {
    "startCislo": 41,
  },
  Object {
    "startCislo": 42,
  },
  Object {
    "startCislo": 43,
  },
  Object {
    "startCislo": 44,
  },
  Object {
    "startCislo": 45,
  },
  Object {
    "startCislo": 46,
  },
  Object {
    "startCislo": 47,
  },
  Object {
    "startCislo": 48,
  },
  Object {
    "startCislo": 49,
  },
  Object {
    "startCislo": 50,
  },
  Object {
    "startCislo": 51,
  },
  Object {
    "startCislo": 52,
  },
  Object {
    "startCislo": 53,
  },
  Object {
    "startCislo": 54,
  },
  Object {
    "startCislo": 55,
  },
  Object {
    "startCislo": 56,
  },
  Object {
    "startCislo": 57,
  },
  Object {
    "startCislo": 58,
  },
  Object {
    "startCislo": 59,
  },
  Object {
    "startCislo": 60,
  },
  Object {
    "startCislo": 61,
  },
  Object {
    "startCislo": 62,
  },
  Object {
    "startCislo": 63,
  },
  Object {
    "startCislo": 64,
  },
  Object {
    "startCislo": 65,
  },
  Object {
    "startCislo": 66,
  },
  Object {
    "startCislo": 67,
  },
  Object {
    "startCislo": 68,
  },
  Object {
    "startCislo": 69,
  },
  Object {
    "startCislo": 70,
  },
  Object {
    "startCislo": 71,
  },
  Object {
    "startCislo": 72,
  },
  Object {
    "startCislo": 73,
  },
  Object {
    "startCislo": 74,
  },
  Object {
    "startCislo": 75,
  },
  Object {
    "startCislo": 76,
  },
  Object {
    "startCislo": 77,
  },
  Object {
    "startCislo": 78,
  },
  Object {
    "startCislo": 79,
  },
  Object {
    "startCislo": 80,
  },
  Object {
    "startCislo": 81,
  },
  Object {
    "startCislo": 82,
  },
  Object {
    "startCislo": 83,
  },
  Object {
    "startCislo": 84,
  },
  Object {
    "startCislo": 85,
  },
  Object {
    "startCislo": 86,
  },
  Object {
    "startCislo": 87,
  },
  Object {
    "startCislo": 88,
  },
  Object {
    "startCislo": 89,
  },
  Object {
    "startCislo": 90,
  },
  Object {
    "startCislo": 91,
  },
  Object {
    "startCislo": 92,
  },
  Object {
    "startCislo": 93,
  },
  Object {
    "startCislo": 94,
  },
  Object {
    "startCislo": 95,
  },
  Object {
    "startCislo": 96,
  },
  Object {
    "startCislo": 97,
  },
  Object {
    "startCislo": 98,
  },
  Object {
    "startCislo": 99,
  },
  Object {
    "startCislo": 100,
  },
]
`;

exports[`maps state to props - půlmaraton, startující, odstartováni 1`] = `
Array [
  Object {
    "cas": "PT2H6M32.6S",
    "dokonceno": true,
    "id": "7a09b1fd371dec1e99b7e142",
    "startCislo": 11,
  },
  Object {
    "dokonceno": false,
    "id": "5a09b1fd371dec1e99b7e1c9",
    "startCislo": 15,
  },
]
`;

exports[`maps state to props - půlmaraton, startující, přihlášky 1`] = `
Array [
  Object {
    "id": "7a09b1fd371dec1e99b7e142",
    "startCislo": 10,
  },
  Object {
    "id": "5a09b1fd371dec1e99b7e1c9",
    "startCislo": 17,
  },
]
`;
